tests/Backend/Bigquery/WorkspacesLoadTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Keboola\Test\Backend\Bigquery;
+
+use Google\Cloud\BigQuery\Table;
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\Backend\Workspaces\Backend\BigqueryWorkspaceBackend;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+use Keboola\Test\Backend\Workspaces\ParallelWorkspacesTestCase;
+use Throwable;
+
+class WorkspacesLoadTest extends ParallelWorkspacesTestCase
+{
+    public function testTableLoadAsView(): void
+    {
+        $bucketId = $this->getTestBucketId(self::STAGE_IN);
+        $workspaces = new Workspaces($this->workspaceSapiClient);
+        $workspace = $workspaces->createWorkspace([], true);
+        $backend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
+        assert($backend instanceof BigqueryWorkspaceBackend);
+        $importFile = __DIR__ . '/../../_data/languages.csv';
+        $tableId = $this->_client->createTableAsync(
+            $bucketId,
+            'languages',
+            new CsvFile($importFile)
+        );
+
+        // test loading alias in IM not supported
+        $aliasId = $this->_client->createAliasTable($bucketId, $tableId, 'languages-alias');
+
+        $options = [
+            'input' => [
+                [
+                    'source' => $aliasId,
+                    'destination' => 'languages',
+                    'useView' => true,
+                ],
+            ],
+        ];
+
+        try {
+            $workspaces->loadWorkspaceData($workspace['id'], $options);
+            $this->fail('Must throw exception as load alias as view is not supported');
+        } catch (ClientException $e) {
+            $this->assertSame('workspace.loadRequestLogicalException', $e->getStringCode());
+            $this->assertSame(
+                'View load is not supported, only table can be loaded using views, alias of table supplied. Use read-only storage instead or copy input mapping if supported.',
+                $e->getMessage()
+            );
+        } catch (Throwable $e) {
```

Comment:
toto tady nemusi byt, resili jsme to tady https://github.com/keboola/storage-api-php-client/pull/1215#discussion_r1386132325 jen jsem si nevsiml vsech vyskytu